adapters/mysql/mysql_test.go

```diff
@@ -3,14 +3,14 @@ package mysql
 import (
 	"database/sql"
 	"fmt"
+	"github.com/doug-martin/goqu"
```

Comment:
Seeing this all over.  Standard library packages should be in their own group.  See run goimports
see https://github.com/golang/go/wiki/CodeReviewComments#imports

Might wanna runn goimports

Reply:
done